docker/compose/docker-compose-user-env.yml

```diff
@@ -0,0 +1,319 @@
+---
+version: '3.7'
+services:
+  nginx:
+    image: mozdef/mozdef_nginx
```

Comment:
Instead of copy/pasting the entire docker-compose.yml file, we can use "inheritance" (docker overrides) here, so that we only have to redefine the env_file location. This could look like:
```
---
version: '3.7'
services:
  nginx:
    env_file:
      - $ENV_FILE
  mongodb:
    env_file:
      - $ENV_FILE
  kibana:
    env_file:
      - $ENV_FILE
  elasticsearch:
    env_file:
      - $ENV_FILE

etc
```

Reply:
Ah indeed that is nicer. I wasn't aware that I could inherit with a docker compose file. Give me a bit and I'll update the file.